controllers/user.js

```diff
@@ -24,17 +11,7 @@ const getUsers = async (_req, res) => {
 const getUser = async (req, res) => {
   const userId = req.params.userId
 
-  const user = await User.findById(userId).populate('role').lean().exec()
-
-  if (!user) throw createError(404, USER_NOT_REGISTERED)
-
-  const userResponse = {
-    id: user._id,
-    firstName: user.firstName,
-    lastName: user.lastName,
-    role: user.role.value,
-    email: user.email
-  }
+  const userResponse = await userService.getUser(userId)
 
   res.status(200).json({
     user: userResponse
```

Comment:
you can call response on 14 line just `user` and use here just `user` instead of `user: userResponse`

Reply:
please, check all similar cases

Reply:
fixed